pkgs/by-name/co/conduwuit/package.nix

```diff
@@ -55,12 +53,7 @@ rustPlatform.buildRustPackage rec {
       zstd
     ]
     ++ lib.optional enableJemalloc rust-jemalloc-sys'
-    ++ lib.optional enableLiburing liburing
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      apple-sdk_15
```

Comment:
Didn't we want to keep the higher sdk versions for building with extra features?

Reply:
Sorry, just read the last part of your description... lmao